docs/source/conf.py

```diff
@@ -441,15 +453,212 @@ def check_timeout_and_api_kwargs_presence(obj: object) -> int:
     )
 
 
+def create_available_in_admonitions() -> dict[str, str]:
+    """Creates a dictionary for 'Available in' admonitions for classes that are available
+    in attributes of other classes.
+    """
+
+    def build_full_class_name(short_class_name: str) -> str:
+        """The keys in the dictionary are not the likes of "telegram.StickerSet"
+        but the likes of "<class 'telegram._files.sticker.StickerSet'>".
+        This function takes a short class name and produces the full name.
+        """
+        return str(eval(short_class_name))
+
+    # First, generate a mapping of class names to the attributes in other classes that correspond
+    # to instances of a given class
+    # i.e. {"telegram.Sticker": ["telegram.Message.sticker", ...]}
+    attrs_for_class_name = defaultdict(list)
+    admonition_for_class_name = {}
+
+    # The colon after the closing parenthesis is important because it makes sure
+    # that the class is mentioned in the attribute description, not in free text.
+    # So far there seem to be no attribute docstrings in which the list of classes
+    # takes up more than one line.
+    attr_docstr_pattern = re.compile(r"^\s*(?P<attr_name>[a-z_]+)\s?\(.*:class:`.+`.*\):\s.*$")
```

Comment:
can you use the `re.VERBOSE` & explain the pattern a bit more? 😬

Reply:
Done